department-of-veterans-affairs/vets-api
# API-40281-sp-issues-pact-previous-rated-disability

## Summary

- Adds validation rule that a special issues should not be labeled as a PACT if the action type is 'increase'.
- Adds test.

## Related issue(s)

- [API-40281](https://jira.devops.va.gov/browse/API-40281)

## Testing done

- Testing notes for Postman
1. Send a disability with the following:
```
                {
                    "name": "Cancer - Musculoskeletal - Knee",
                    "exposureOrEventOrInjury": "EXPOSURE",
                    "approximateDate": "2015",
                    "serviceRelevance": "ABCDEFG",
                    "disabilityActionType": "INCREASE",
                    "classificationCode": "9014",
                    "ratedDisabilityId": "ABCDEFGHIJKLMNOPQRSTUVWX",
                    "diagnosticCode": 249470,
                    "isRelatedToToxicExposure": true
                }
```
2. place a debugger at the end of the evss mapper to check if the special issues includes 'PACT' for the disability that has an action type as 'increase'

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
	modified:   modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb
	modified:   modules/claims_api/spec/lib/claims_api/v2/disability_compensation_evss_mapper_spec.rb

## Acceptance criteria
